haskell/test/Lorentz/Contracts/Test/Permit.hs

```diff
@@ -304,28 +307,26 @@ permitSpec originate = do
               lCallEP stablecoinContract (Call @"Revoke") [RevokeParam hash testPauser]
 
     describe "Set_expiry" $ do
```

Comment:
Can you please add a test: "setting the expiry of an already expired permit does not long prolong its life"

Reply:
@dcastro This check was missing in my original changes. I have added it and the test for it now.